articles/techniques/testing-and-debugging.md

```diff
@@ -304,19 +304,19 @@ Wavefunction:
 1:      -0.5            -0.5
 ```
 
-In general, the state of a register that is entangled with another register is a mixed state rather than a pure state. In this case, @"microsoft.quantum.extensions.diagnostics.dumpregister" outputs the following message:
+In general, the state of a register that is entangled with another register is a mixed state rather than a pure state. In this case, <xref:microsoft.quantum.extensions.diagnostics.dumpregister> outputs the following message:
```

Comment:
```suggestion
In general, the state of a register that is entangled with another register is a mixed state rather than a pure state. In this case, <xref:microsoft.quantum.diagnostics.dumpregister> outputs the following message:
```